tests/out/agency.txt

```diff
@@ -1,31 +1,31 @@
-agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone,agency_fare_url,agency_email
--1,Unknown agency,,,,,,
-4406935663585698503,BSAG,,Europe/Berlin,,,,
-302399630421482339,DELBUS,,Europe/Berlin,,,,
-7768149551816981722,BremerhavenBus,,Europe/Berlin,,,,
-5219358392088886743,Masemann,,Europe/Berlin,,,,
-2358389688163451390,Weser-Ems_Bus,,Europe/Berlin,,,,
-5094654536138210010,DB Fernverkehr AG,,Europe/Berlin,,,,
-3161766525275180161,NordWestBahn GmbH,,Europe/Berlin,,,,
-1691986616946700729,DHE,,Europe/Berlin,,,,
-8778963060538961469,Weser-Ems Busverkehr GmbH,,Europe/Berlin,,,,
-3775939454649125116,Weser-Ems-Bus,,Europe/Berlin,,,,
-3506382908292887946,Weser-Ems Bus,,Europe/Berlin,,,,
-4259177160148784091,EVB,,Europe/Berlin,,,,
-1723010500960325508,NordWestBahn,,Europe/Berlin,,,,
-6396067022764612297,Deutsche Bahn,,Europe/Berlin,,,,
-7136420037655560711,erixx,,Europe/Berlin,,,,
-3726107220948082555,Giese,,Europe/Berlin,,,,
-7065487351021650981,Mein Fernbus,,Europe/Berlin,,,,
-8451440369930911025,Schweizerische Bundesbahnen;DB Fernverkehr AG,,Europe/Berlin,,,,
-5816593794701784995,v. Rahden,,Europe/Berlin,,,,
-3467737500187716770,DHE/Wolters,,Europe/Berlin,,,,
-7636439714841453962,Weser-Ems Bus (RL Bremen),,Europe/Berlin,,,,
-1091007832018125634,Weser-Ems-Bus & Buschmann,,,,,,
-7539215502515416313,DB Regio Nord,,Europe/Berlin,,,,
-3419539873666858620,Hamburg-Köln-Express GmbH,,Europe/Berlin,,,,
-4288964907121629380,Bürgerbus Ritterhude e.V.,,Europe/Berlin,,,,
-7096273444620035959,Flixbus,,Europe/Berlin,,,,
-7015522928844581004,"WEB, VGB",,Europe/Berlin,,,,
-7905431934736606341,DB Regio AG,,Europe/Berlin,,,,
-3945656614886157833,DB,,Europe/Berlin,,,,
+agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone,agency_fare_url,agency_email
+-1,Unknown agency,,,,,,
+7525367854195739796,BSAG,,Europe/Berlin,,,,
```

Comment:
well this is very interesting ! It seems that having these files is already paying off :)
Why did the agency id change? I would expect the same the hashing function to return the same agency id for every run :thinking:

Reply:
@carocad good catch! Apparently it changes every time, I'm working on that.

Reply:
@carocad Please see my changes. The problem was the `hash` function in python 3 only produces unique hashes in a process not over different runs. Instead of using a hash I removed it all together. The `agency_name` doesn't need to be an integer and it is more readable when we a name instead.